source/includes/hourly-ride-api.md

```
@@ -0,0 +1,699 @@
+
+| Parameter       | Type    | Description                                                                                                                                                                |
+| --------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| originType      | string  | Specifies whether the origin is provided as geo-coordinates or as an airport code. Possible values: `coordinates`, `iata`. If omitted, the default value is `coordinates`. |
+| originLatitude  | number  | Pickup location latitude in degrees. Required if `originType` is set to `coordinates` or omitted.                                                                          |
+| originLongitude | number  | Pickup location longitude in degrees. Required if `originType` is set to `coordinates` or omitted.                                                                         |
+| origin          | string  | IATA airport code of the pickup airport. Required if `originType` is set to `iata`.                                                                                        |
+| departureTime   | integer | Departure time as a UNIX epoch timestamp in seconds. This timestamp should be calculated from the local departure time and converted to UTC.                               |
+| hoursRented     | integer | Duration in hours (1-24). Required. The service includes 20 km of travel per hour.                                                                                         |
+| passengersCount | integer | Total number of passengers to transport (adults and children). Must be between 1 and 7.                                                                                    |
+| childrenCount   | integer | Optional. Specifies the number of children in the group.                                                                                                                   |
+|                 |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 3; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)